src/tasks/AndroidAppBuilder/Templates/monodroid-coreclr.c

```diff
@@ -24,12 +24,21 @@ void
 Java_net_dot_MonoRunner_setEnv (JNIEnv* env, jobject thiz, jstring j_key, jstring j_value);
 
 int
-Java_net_dot_MonoRunner_initRuntime (JNIEnv* env, jobject thiz, jstring j_files_dir, jstring j_cache_dir, jstring j_testresults_dir, jstring j_entryPointLibName, jobjectArray j_args, long current_local_time);
+Java_net_dot_MonoRunner_initRuntime (JNIEnv* env, jobject thiz, jstring j_files_dir, jstring j_cache_dir, jstring j_testresults_dir, jstring j_entryPointLibName, long current_local_time);
+
+int
+Java_net_dot_MonoRunner_execEntryPoint (JNIEnv* env, jobject thiz, jstring j_entryPointLibName, jobjectArray j_args);
+
+void
+Java_net_dot_MonoRunner_freeNativeResources (JNIEnv* env, jobject thiz);
 
 /********* implementation *********/
 
-static char *bundle_path;
-static char *executable;
+static char* g_bundle_path = NULL;
+static char* g_executable_path = NULL;
```

Comment:
```suggestion
static const char* g_bundle_path = NULL;
static const char* g_executable_path = NULL;
```

Reply:
Thanks, edited for both Mono and CoreCLR.